man/signal-cli.1.adoc

```diff
@@ -208,6 +208,19 @@ number::
 	Specify the safety number or fingerprint of the key, only use this option if you have verified
 	the fingerprint.
 
+setContactName
+--------------
+Update name associated to a number on our contact list. This change is only local but can be synchronized to other devices by using `sendContacts` (see below).
+
+number::
+	Specify the contact phone number.
+
+name::
+	Specify the new name for this contact.
+
+sendContacts
+------------
+Update contact list on Signal servers.
```

Comment:
This description is not really correct. The Signal servers don't store the contacts list. They just forward the encrypted contacts list to the linked devices.

Suggestion:
`Send a synchronization message with the local contacts list to all linked devices. This command should only be used if this is the master device.`

Please also update the help text in `SendContactsCommand`.